index.html

```diff
@@ -0,0 +1,2199 @@
+          </div>
+        </div>
+      </section>
+
+      <div class="d-sm-block d-md-none d-lg-none">
+        <article class="card rounded-0">
+          <div>
+            <img class="rounded-0 w-100 border-0 border-top-0" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="President Trump">
+            <a href="#">
+              <p class="politics">POLITICS</p>
+            </a>
+          </div>
+          <div class="card-body">
+            <a href="#">
+              <h2 class="mag_link2 article-title pt-3">
+                COVID-19 Linked to Changes to the Brains of Some Patients
+              </h2>
+            </a>
+            <p class="card-text">
+              Have the pollsters learned from their mistakes in the 
+              2016 presidential election? Maybe. Maybe not
+            </p>
+          </div>
+        </article>
+      </div>
+
+      <div class="d-sm-block d-md-none d-lg-none my-3">
+        <article class="card rounded-0">
+          <div>
+            <img class="rounded-0 w-100 border-0 border-top-0" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="President Trump">
+            <a href="#">
+              <p class="politics">POLITICS</p>
+            </a>
+          </div>
+          <div class="card-body">
+            <a href="#">
+              <h2 class="mag_link2 article-title pt-3">
+                COVID-19 Linked to Changes to the Brains of Some Patients
+              </h2>
+            </a>
+            <p class="card-text">
+              Have the pollsters learned from their mistakes in the 
+              2016 presidential election? Maybe. Maybe not
+            </p>
+          </div>
+        </article>
+      </div>
+
+      <div class="d-sm-block d-md-none d-lg-none my-3">
+        <article class="card rounded-0">
+          <div>
+            <img class="rounded-0 w-100 border-0 border-top-0" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="President Trump">
+            <a href="#">
+              <p class="politics">POLITICS</p>
+            </a>
+          </div>
+          <div class="card-body">
+            <a href="#">
+              <h2 class="mag_link2 article-title pt-3">
+                COVID-19 Linked to Changes to the Brains of Some Patients
+              </h2>
+            </a>
+            <p class="card-text">
+              Have the pollsters learned from their mistakes in the 
+              2016 presidential election? Maybe. Maybe not
+            </p>
+          </div>
+        </article>
+      </div>
+
+      <section class="more-stories d-md-none d-lg-none mt-5">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder border-bottom">MORE STORIES</h2>
+        </a>
+        <article class="more-stories-article d-flex border-bottom">
```

Comment:
On the mobile version, the articles on this section are less wider than the original page. Please make them look like the original page:
![Screenshot from 2020-04-22 13-35-58](https://user-images.githubusercontent.com/51926937/80020198-3d844800-849e-11ea-94dc-0fd72802137c.png)


Original:
![Screenshot from 2020-04-22 13-36-06](https://user-images.githubusercontent.com/51926937/80020208-4117cf00-849e-11ea-90eb-85d4f85b4703.png)